scheduler/src/main/java/org/pentaho/platform/scheduler2/quartz/BlockingQuartzJob.java

```diff
@@ -50,6 +50,9 @@ public void execute( final JobExecutionContext jobExecutionContext ) throws JobE
             "Job '" + jobExecutionContext.getJobDetail().getName()
                 + "' attempted to run during a blockout period.  This job was not executed" );
       }
+    } catch ( ActionAdapterQuartzJob.LoggingJobExecutionException le ) {
+      // thrown by the execution code - if execution fails, there's nothing we can do, no point in trying to execute
```

Comment:
We already have log entries internal to the code that would cause the exceptions - this is purely to prevent a job retry